src/main/kotlin/racing/domain/Cars.kt

```diff
@@ -1,21 +1,16 @@
 package racing.domain
 
-class Cars(count: Int) : Iterable<Car> {
+class Cars(list: List<Car>) : Iterable<Car> {
```

Comment:
```suggestion
class Cars(private val list: List<Car>) : Iterable<Car> {
```
코틀린은 주생성자에서 프로퍼티의 선언까지 할 수 있습니다 😄 
코드가 조금 더 간결해지는 효과가 있겠네요.

Reply:
넵!